Sources/Modules/Yggdrasil/YggdrasilTown/NPCs/NPCSpawnManager.cs

```diff
@@ -14,18 +14,50 @@ public override void EditSpawnPool(IDictionary<int, float> pool, NPCSpawnInfo sp
 		}
 
 		var dict = new Dictionary<int, float>(yggdrasilNPC.Count);
-		foreach(var pair in pool)
+		foreach (var pair in pool)
 		{
-			if(yggdrasilNPC.Contains(pair.Key))
+			if (yggdrasilNPC.Contains(pair.Key))
 			{
 				dict.Add(pair.Key, pair.Value);
 			}
 		}
 		pool.Clear();
-		foreach(var pair in dict)
+		foreach (var pair in dict)
 		{
 			pool.Add(pair.Key, pair.Value);
+
+		}
+		pool.Add(NPCID.BlueSlime, 0.1f);
+		pool.Add(NPCID.GreenSlime, 0.1f);
+		pool.Add(NPCID.RedSlime, 0.1f);
+		pool.Add(NPCID.PurpleSlime, 0.1f);
+		pool.Add(NPCID.YellowSlime, 0.1f);
+		pool.Add(NPCID.BlackSlime, 0.1f);
+		pool.Add(NPCID.GoldenSlime, 0.1f);
+		pool.Add(NPCID.MotherSlime, 0.1f);
+		pool.Add(NPCID.Pinky, 0.1f);
+		if (Main.halloween)
+		{
+			pool.Add(NPCID.Pinky, 0.1f);
+		}
```

Comment:
I'm not sure why the pinky spawn rates are the same for halloween. Did you mean to make it spawn more commonly during halloween?

Reply:
sorry，it‘s just a mistake 😢